crates/subspace-farmer/src/bin/subspace-farmer/commands/farm.rs

```diff
@@ -664,6 +664,11 @@ where
 
     info!("Finished collecting already plotted pieces successfully");
 
+    for (_, single_disk_farm) in single_disk_farms.iter().enumerate() {
+        farmer_metrics.update_farm_size(single_disk_farm.id(), single_disk_farm.total_sectors_count() as i64);
```

Comment:
But i64 is already used in the project on the interface of the established_connections gauge. Should I use the same gauge type and convert values in the updating functions?

Reply:
Yes, that is what I would prefer. We try to use domain-specific types in various places if possible and convert to the actual types of underlying libraries in case they mismatch at the boundaries.

Reply:
Is it OK that single_disk_farm.plotted_sectors_count() and single_disk_farm.total_sectors_count() return different types?

Reply:
There are some technical reasons for it, will likely change in the future to the same type

Reply:
Is it OK to use .try_into().unwrap() for converting in that case?

Reply:
I would prefer `as` in this particular instance and we don't use `.unwrap()` anywhere except tests as contribution guidelines suggest.

Reply:
So, should I change now?

Reply:
I'll update it in the upcoming PR